packages/one-app-runner/bin/one-app-runner-test.js

```diff
@@ -15,29 +15,64 @@
  */
 
 require('dotenv').config();
-const { spawn } = require('child_process');
-const { waitForOK } = require('../utils/waitForOK');
+const { spawn } = require('node:child_process');
```

Comment:
What is the difference `node:child_process` vs `child_process?

Reply:
the `node` prefix e.g. `node:child_process` is a convention to help identify what packages are natively provided and what need to be installed